libnmstate/netinfo.py

```diff
@@ -91,6 +96,30 @@ def _devinfo_bond(dev, devinfo):
             }
     return {}
 
+def _devinfo_ip(dev, devinfo):
+    ip4info = {}
+    connection = dev.get_active_connection()
```

Comment:
Please move all NM related details to nm.* , here we should only be concern with the higher format,
Something like this:
```
def _devinfo_ipv4(dev):
    act_con = nm.connection.get_device_active_connection(dev)
    return {'ipv4': nm.ipv4.get_state(act_con)}
```